src/utils/TraceUtils.js

```diff
@@ -177,3 +170,73 @@ export const treeDecorate = (key, f, tree) => {
     newTree[key] = f(tree);
     return newTree;
 }
+
+export const highlightProperty = 'isHighlighted';
+
+// Takes a tree and attaches the property 'isHighlighted' to every node in the
+// tree. All those nodes whose trace has the given trace as prefix will get
+// 'isHiglighted' = true, and the rest false.
+export const forwardMatching = (trace, tree) => {
+    let f;
+    if (trace === null) {
+        // Completely null traces do not match anything.
+        f = (t) => (false);
```

Comment:
Yes, that is true, and I was aware of it. :smiley: As a personal preference of style, I like to have parentheses around the arguments of an arrow function regardless of how many there are -- to me, it emphasizes that "this is a function and this is its argument", since I think they blend together a bit otherwise.